assets/sass/components/_menu-notification.scss

```diff
@@ -0,0 +1,127 @@
+.menu-notification .glyphicon {
+	margin-right: 18px;
+}
+.menu-notification .badge {
+	position: absolute;
+	left: 28px;
+	top: 15px;
+	-webkit-animation: bounce 10s ease-in-out;
+	-moz-animation: bounce 10s ease-in-out;
+	-o-animation: bounce 10s ease-in-out;
+	animation: bounce 10s ease-in-out;
+}
+@-webkit-keyframes bounce {
```

Comment:
FYI: this might be a good place for a mixing. See https://stackoverflow.com/a/33203668